packages/kbn-optimizer/src/node/cache.ts

```diff
@@ -101,28 +101,27 @@ export class Cache {
     }
   }
 
-  async update(path: string, file: { mtime: string; code: string; map: any }) {
-    const key = this.getKey(path);
+  close() {
+    clearTimeout(this.timer);
+  }
 
+  async update(path: string, file: { mtime: string; code: string; map?: any }) {
+    const key = this.getKey(path);
     await Promise.all([
       this.safePut(this.atimes, key, GLOBAL_ATIME),
       this.safePut(this.mtimes, key, file.mtime),
       this.safePut(this.codes, key, file.code),
-      this.safePut(this.sourceMaps, key, JSON.stringify(file.map)),
+      file.map != null ? this.safePut(this.sourceMaps, key, JSON.stringify(file.map)) : null,
```

Comment:
Don't we have to update the getSourcemap function too?

Reply:
I don't think so, that function has always been prepared for there not being a source map value for a file in the cache